src/workerd/util/sqlite.h

```diff
@@ -19,6 +20,20 @@ namespace workerd {
 using kj::byte;
 using kj::uint;
 
+class SqliteObserver {
+public:
+  virtual void addRowsRead(uint64_t rowsRead) {}
+  virtual void addRowsWritten(uint64_t rowsWritten) {}
+  virtual void setSrsStoredBytesCallback(kj::Function<uint64_t()> callback) {}
```

Comment:
I think `setSrsStoredBytesCallback()` could be part of `ActorObserver` or maybe even `InternalActorObserver` (in the internal codebase), since it is not actually called by `SqliteDatabase` or anywhere in workerd.